### ISSUE TITLE
Delays in appearance of tracker content

### ISSUE THREAD

**Issue by darkcaldera** [pyrochlore/obsidian-tracker#33]
Originally opened this to ask about why I couldn't get the month view trackers to appear. Then, later on, they started working, and it wasn't because of anything I'd changed in my setup. (I did change things, but changing them back to test didn't break the month view.)

I'm still having issues with not all of my entries showing up on bar and line graphs, even for ones written and detailed in the exact same way as a working graph, with the *only* difference being the searchTarget. All the bits it's looking for are in the same format and in the same note as an entry that is working for a different searchTarget (And it's not as if the new searchTarget itself can find nothing, as some of the instances are appearing. But others with identical format, pasted in a different Daily Note, are not appearing.) Perhaps this is also a delay, but some of it has been missing for longer than the month view trackers were.

All-round weird experience with this. But it looks really handy, so I really hope I can straighten it out!

**Comment by pyrochlore** [comment]
Hi, thanks for the feedback.
Would you mind to give me a simplified example showing the weird behavior you met.

I tested the output of this plugin every time before release using the examples in [example folder](https://github.com/pyrochlore/obsidian-tracker/tree/master/examples), not met things like you mentioned.
It would be helpful if I can have some examples to do the test.

**Comment by darkcaldera** [comment]
Okay, I'm unable to reproduce the issue of the calendar/month view not loading at all, since it seems to have started behaving now. As such I can't *swear* to the process, as I wasn't screen-recording, but as far as I can recall, the following arrangement wasn't working, and then, after messing with some bar and line views (separate code blocks but on the same page), it was.

``` tracker
searchType: dvField
searchTarget: Hydration
folder:  /· Diary/Daily Notes
month:
    startWeekOn: Mon
```

To go with this are values in mL at the bottom of several of my daily notes to test it, formatted ``` Hydration:: 2000mL ``` or some other number. It's working now and I don't know why exactly, sorry.

I currently don't have any graphs that aren't appearing, but some data isn't loading in any version of a graph tracking daily push-ups. This is the tracker I've written:

``` tracker
searchType: dvField
searchTarget: Push-Ups
folder: /· Diary/Daily Notes
month:
    startWeekOn: Mon
```

Very similar to the Hydration one, although for some reason *only* the number from the 29th (today) is appearing. The test trackers on that day in the Daily Note read:

```
Hydration:: 1750mL
Push-Ups:: 40
```

And this is coming through. But for the previous day, where the test figures in the Daily Note read

```
Hydration:: 2000mL
Push-Ups:: 38
```

The Hydration value for this day is being correctly added to its month view, while the Push-Up value is not. I... don't understand what could be causing this difference.

I hope that's useful information, I'm fairly amateur at this sort of thing. But having fun with it! Very impressive plugin :)

**Comment by pyrochlore** [comment]
Thanks for the sharing.
I know sometimes it's hard to identify or repeat where the bugs is.
I will make an example based on your information and post here later.
Then you can compare it to yours.

**Comment by pyrochlore** [comment]
> I hope that's useful information, I'm fairly amateur at this sort of thing. But having fun with it! Very impressive plugin :)

Very useful information.

Found a bug that would lead to missing values if dataview inline fields were placed at the last line of a file. 
You may add blank lines to your notes after those fields as a temporary solution.
The bug will be fixed and released in the next release.

**Comment by darkcaldera** [comment]
Oh, great!

You're completely right, just adding a blank line at the bottom of those daily notes fixed the issue. Would never have thought of that! (In hindsight, I probably should have checked if changing the order of those tracked fields made a difference, but you were on it.)

Awesome work :D

**Comment by pyrochlore** [comment]
The fix of this bug was just released in version 1.8.0.
Enjoy tracking.

**Comment by pyrochlore** [comment]
Issue closed.
Welcome to fire others for bugs or feature requests.
